model_analyzer/config/input/config_command.py

```diff
@@ -325,6 +327,26 @@ def get_all_config(self):
 
         return config_dict
 
+    def set_constraints_holder(self):
+        """
+        Used for creating a Constraints Holder object
+        
+        Default Parameters
+        ----------
+        self : ConfigCommandProfile/ConfigCommandReport based on the child class inherited
+        """
+        self._constraints_holder = ConstraintsHolder(self)
```

Comment:
So we have to create a config object, and give it an empty ConstraintsHolder. And then we need to tell the ConstraintsHolder to initialize itself from the config object that it is already inside of? This is not straight forward or clean.  I think the clean way to do this is to put a single model constraint class inside of ProfileModels. Then it is already passed everywhere we need it and nothing about the CLI/config code needs to change.